tools/wptrunner/wptrunner/wptmanifest/serializer.py

```diff
@@ -1,3 +1,5 @@
+from six import ensure_str, ensure_text
```

Comment:
Looks like we need to upgrade `six` to get this.

Reply:
Yes, i was looking at the same thing. What file do i have to edit?

Reply:
It requires a `six` version `>=1.12.0` but I can see `six==1.13.0` in `tools/wptrunner/requirements.txt`.
Do i have to also specify it in all per browser requirements_<browser>.txt files?

Reply:
The issue is that our vendored copy is out of date. I'm investigating how to update it without breaking the world; it looks like something strange happened with the import.

Reply:
I just merged the PR to update six, I'll retrigger this once the other CI failures are fixed; then if it passes I think we're good.

Reply:
Should I rebase?

Reply:
I think just close+reopen should work since we test the provisional merge commit into master.